packages/ember-testing/lib/helpers.js

```diff
@@ -111,8 +111,16 @@ asyncHelper('fillIn', fillIn);
   var $el = find('.my-selector');
   ```
 
+  With the `context` param:
+
+  ```javascript
+  var $el = find('.my-selector', 'parent-element');
```

Comment:
Ah, my bad, quick PR and didn't check it through, thank you for pointing that out! :)